alvr/client_openxr/src/extra_extensions/motion_tracking_bd.rs

```diff
@@ -96,8 +92,7 @@ impl MotionTrackerBD {
         let mut locations = Vec::with_capacity(3);
 
         let mut connect_state = MotionTrackerConnectStateBD {
-            tracker_count: 0,
-            serials: [MotionTrackerSerialBD { serial: [0; 24] }; 6],
+            ..Default::default()
```

Comment:
This can be further simplified as `MotionTrackerConnectStateBD::default()`

Reply:
Already did locally